README.md

```diff
@@ -1,27 +1,45 @@
-# type-detect [![Build Status](https://travis-ci.org/chaijs/type-detect.png?branch=master)](https://travis-ci.org/chaijs/type-detect) [![Coverage Status](https://coveralls.io/repos/chaijs/type-detect/badge.png?branch=master)](https://coveralls.io/r/chaijs/type-detect?branch=master)
+<h1 align=center>[![Chai Documentation](http://chaijs.com/img/chai-logo.png)](http://chaijs.com) type-detect</h1>
 
-> Improved typeof detection for node.js and the browser.
+<p align=center>
+  Improved typeof detection for [node](http://nodejs.org) and the browser.
+</p>
+
+<p align=center>
+  [![license:mit](https://img.shields.io/badge/license-mit-green.svg?style=flat-square)](#license)
+  [![tag:?](https://img.shields.io/github/tag/chaijs/type-detect.svg?style=flat-square)](https://github.com/chaijs/type-detect/releases)
+  [![build:?](https://img.shields.io/travis/chaijs/type-detect/master.svg?style=flat-square)](https://travis-ci.org/chaijs/type-detect)
+  [![coverage:?](https://img.shields.io/coveralls/chaijs/type-detect/master.svg?style=flat-square)](https://coveralls.io/r/chaijs/type-detect)
+  [![npm:?](https://img.shields.io/npm/v/type-detect.svg?style=flat-square)](https://www.npmjs.com/packages/type-detect)
+  [![dependencies:?](https://img.shields.io/npm/dm/chai.svg?style=flat-square)](https://www.npmjs.com/packages/chai)
+  [![devDependencies:?](https://img.shields.io/david/chaijs/type-detect.svg?style=flat-square)](https://david-dm.org/chaijs/type-detect)<br>
+  [![Selenium Test Status](https://saucelabs.com/browser-matrix/chaijs-type-detect.svg)](https://saucelabs.com/u/chaijs-type-detect)<br>
+  [![Join the Slack chat](https://img.shields.io/badge/slack-join%20chat-E2206F.svg?style=flat-square)](https://chai-slack.herokuapp.com/)
+  [![Join the Gitter chat](https://img.shields.io/badge/gitter-join%20chat-D0104D.svg?style=flat-square)](https://gitter.im/chaijs/type-detect)
+</p>
+
+## What is Type-Detect?
+
+Type Detect is a module which you can use to detect the type of a given object. It returns a string representation of the object's type, either using [`typeof`](http://www.ecma-international.org/ecma-262/6.0/index.html#sec-typeof-operator) or [`@@toStringTag`](http://www.ecma-international.org/ecma-262/6.0/index.html#sec-symbol.tostringtag). It also normalises some object names for consitency among browsers.
```

Comment:
**It also normalises some object names for consitency among browsers.**
A little typo here on the word "consistency"